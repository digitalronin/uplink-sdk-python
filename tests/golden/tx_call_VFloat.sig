r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 46799189056555781432434637613908364642278716971883109259051124473258479017610
