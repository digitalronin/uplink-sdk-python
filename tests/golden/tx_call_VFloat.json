{
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "header": {
        "tag": "TxContract", 
        "contents": {
            "tag": "Call", 
            "contents": {
                "args": [
                    {
                        "tag": "VFloat", 
                        "contents": 3.5
                    }
                ], 
                "method": "get", 
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve"
            }
        }
    }, 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE0zNzE4MDgzNzM2MTU5MzQxMjAxMzEzNjM2Mjc4MTYwNTcyMzYwMTU3MDkzNDQ4MTM5OTg2NDI4OTMwOTAxNzk3MjM4OTkyMDAxNDY5MQ==", 
    "timestamp": 1231006505
}